util/reggen/clocking.py

```diff
@@ -16,6 +16,7 @@ def __init__(self,
                  reset: Optional[str],
                  idle: Optional[str],
                  primary: bool,
+                 internal: Optional[bool],
```

Comment:
I think this is just `bool`, right? (It's optional in the json schema, but we default to `False` in the call to `check_bool` below).

Reply:
yeah you're right, updated.